src/meta/processors/partsMan/CreateSpaceProcessor.cpp

```diff
@@ -119,6 +137,22 @@ CreateSpaceProcessor::pickHosts(PartitionID partId,
     return pickedHosts;
 }
 
+int32_t CreateSpaceProcessor::getSpaceSum() {
+    folly::SharedMutex::ReadHolder rHolder(LockUtils::spaceLock());
+    int32_t count = 0; 
+    auto prefix = MetaServiceUtils::spacePrefix();
+    std::unique_ptr<kvstore::KVIterator> iter;
+    auto ret = kvstore_->prefix(kDefaultSpaceId, kDefaultPartId, prefix, &iter);
+    if (ret != kvstore::ResultCode::SUCCEEDED) {
+        return -1;
+    }
+    std::vector<cpp2::IdName> spaces;
```

Comment:
It seems an unuseful variables: `spaces`

Reply:
Good point, I will carefully rewrite this pr